src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -84,35 +106,42 @@ public OSIsoftServer() {}
    * @param Headers Request headers
    * @param TextFields Request Payload
    * @param FileName File name for response
-   * @return The HTTP response code
+   * @return True/False message was successfully sent
    */
   public static boolean RequestHTTPS(
-      String CnxParam,
-      String Method,
-      String Headers,
-      String TextFields,
-      String FileName) {
-    int res = SCHttpUtility.HTTPX_CODE_NO_ERROR;
+      String CnxParam, String Method, String Headers, String TextFields, String FileName) {

```

Comment:
More of a suggestion since these parameters were not changed within the PR, but the naming convention should be updated or logged to be updated. Capital letter for first character is not permitted per code style convention.

Reply:
will make an issue